src/app/components/turnos/dashboard/estadisticas-pacientes.component.ts

```diff
@@ -69,6 +73,15 @@ export class EstadisticasPacientesComponent implements OnInit {
             nroCarpeta: ''
         };
         this.getPaciente();
+        this.loadObraSocial();
```

Comment:
El código fechaDesde: moment().startOf('month').format() --> this.fechaDesde = new Date(hoy.fechaDesde) habría que cambiarlo....

Lo apropiado sería....
this.fechaDesde = moment().startOf('month').toDate()
y lo mismo para fechaHasta